src/sssom/constants.py

```diff
@@ -271,3 +272,41 @@ def _get_sssom_schema_object() -> SSSOMSchemaView:
 
 SSSOM_URI_PREFIX = "https://w3id.org/sssom/"
 DEFAULT_LICENSE = f"{SSSOM_URI_PREFIX}license/unspecified"
+
+#: The type for metadata that gets passed around in many places
+MetadataType = Dict[str, Any]
+
+
+def generate_mapping_set_id() -> str:
```

Comment:
Is it ok to have methods in the `constants` file?

Reply:
I see the inconsistency here, but I think it outweights the maze of imports you have to go through to find things to keep these closely related parts together.

Reply:
Ok!